src/main/java/com/Ultramega/CentrifugeTiersReproduced/blocks/TieredCentrifuge.java

```diff
@@ -53,4 +61,72 @@ private BlockEntityType getBlockEntity() {
             default -> { return null; }
         }
     }
+
+    @Override
+    public void appendHoverText(@NotNull ItemStack stack, @javax.annotation.Nullable BlockGetter getter, @NotNull List<Component> components, @NotNull TooltipFlag flag) {
+        super.appendHoverText(stack, getter, components, flag);
+        if(Screen.hasShiftDown()) {
+            int outputMultiplier;
+            int speedMultiplier;
+            int additionalCombSlots;
+            int maxStackSize;
+            int energyFluidCapacity;
+
+            switch (this.tier) {
```

Comment:
These variables with the switch statement are not required you can instead just use tier.getSpeed(), tier.getItemMaxStackSize() etc.